tools/utils/tasks_tools.ts

```diff
@@ -9,14 +9,15 @@ import {TOOLS_DIR} from '../config';
 
 const TASKS_PATH = join(TOOLS_DIR, 'tasks');
 
-// NOTE: Remove if no issues with runSequence function below.
-// export function loadTasks(): void {
-//   scanDir(TASKS_PATH, (taskname) => registerTask(taskname));
-// }
+export function loadTasks(): void {
+  scanDir(TASKS_PATH, (taskname) => registerTask(taskname));
+}
 
 export function task(taskname: string, option?: string) {
-  util.log('Loading task', chalk.yellow(taskname, option || ''));
-  return require(join('..', 'tasks', taskname))(gulp, gulpLoadPlugins(), option);
+  return () => {
+    util.log('Loading task', chalk.yellow(taskname, option || ''));
+    return require(join('..', 'tasks', taskname))(gulp, gulpLoadPlugins(), option)();
+  };
 }
 
 export function runSequence(...sequence: any[]) {
```

Comment:
Yes i think its useless for now also if u think about that, if u call the function with (x,y,z) if z is invalid task name it will still work.. it will just ignore it and run what it actually finds. So i remove your method.